package.json

```diff
@@ -105,7 +105,7 @@
     "@gravity-ui/prettier-config": "^1.0.1",
     "@gravity-ui/stylelint-config": "^1.0.1",
     "@gravity-ui/tsconfig": "^1.0.0",
-    "@gravity-ui/uikit": "^3.20.2",
+    "@gravity-ui/uikit": "^4.10.0",
```

Comment:
It seems you should also update `navigation` package, because 0.3.1 version requires 3-rd uikit major.